sdk/purview/Microsoft.Azure.Management.Purview/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "purview/resource-manager" -AutoRestVersion "v2" -ConfigFileTag "package-2020-12-01-preview"
```

Comment:
Please remove `-ConfigFileTag` and rely on the readme.md default tag. Your swagger readme.md is pointing to the correct package. So remove it here. Use that to control the package tag instead of here. Generate.ps1 can remain as unchanged going forward.